### PR TITLE
Adnuntius: Read device information from ortb Request

### PR DESCRIPTION
**Pull request by mikael-lundin**
This PR would make it possible for the requesting source to send device information to our adserver that would be used for geo-targeting and device targeting.

### CODE REVIEW

File: adapters/adnuntius/adnuntiustest/supplemental/send-header-information.json [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+                    }
+                }
+            }
+        ]
+    },
+
+    "httpCalls": [
+        {
+            "expectedRequest": {
+                "uri": "http://whatever.url?format=json&tzo=0",
```
Comment (VeronikaSolovei9) [inline adapters/adnuntius/adnuntiustest/supplemental/send-header-information.json:41]: Please add a check for expected headers under under `httpCalls.expectedRequest`:

```
"headers": {
          "Content-Type": [
            "application/json;charset=utf-8"
          ],
          "Accept": [
            "application/json"
          ],
          "X-Forwarded-For": [
            "158.174.81.143"
          ],
          "User-Agent": [
            "Mozilla/5.0 (Linux; Android 8.0; Pixel 2 Build/OPD3.170816.012) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.93 Mobile Safari/537.36"
          ]
        },

```

Comment (VeronikaSolovei9) [reply adapters/adnuntius/adnuntiustest/supplemental/send-header-information.json:41]:
```
"httpCalls": [
    {
      "expectedRequest": {
           "headers": {
                .....   
           },
          "uri": "http://whatever.url?format=json&tzo=0",
```

Comment (mikael-lundin) [reply adapters/adnuntius/adnuntiustest/supplemental/send-header-information.json:41]: Thanks! I was looking for how to add that but I couldn't really find it anywhere. I've now updated the coed.

**Review by VeronikaSolovei9 — APPROVED** [review]
Thank you for addressing my comment, LGTM

**Review by AlexBVolcy — APPROVED** [review]

**Pull request merged**